source/common/i18n.js

```diff
@@ -102,7 +102,7 @@ export function trans (string, ...args) {
     } else {
       // Something went wrong and the requested translation string was
       // not found -> fall back and just return the original string
-      return (Boolean(global.config.get('debug')) || skipFallback) ? string : trans(string, ...[true].concat(args))
+      return (Boolean(global.config && global.config.get('debug')) || skipFallback) ? string : trans(string, ...[true].concat(args))
```

Comment:
Even though this looks like a bug, this change is not related to #6 or #9. I suggest don't change `source/common/i18n.js`. If you want to fix bug the bug, raise a bug report with [Zettlr/Zettlr](https://github.com/Zettlr/Zettlr)